### PR TITLE
History card updated

### PR DESCRIPTION
**Pull request by Irma12**
This updates Readme for history card introduced lately and also adds some verification for older ArgoCD versions not supporting deployedAt. 

https://github.com/argoproj/argo-cd/issues/3756

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -15,7 +15,6 @@ export const argoCDAppDetails = t.type({
     history: t.array(t.type({
       id: t.number,
       revision: t.string,
-      deployStartedAt: t.string,
       deployedAt: t.string,
```
Comment (iain-b) [inline src/types.ts:18]: I'm wondering how this io-ts library will handle the case where deployedAt is undefined. Will it fail trying to decode?

Comment (Irma12) [reply src/types.ts:18]: @iain-b Think you got a point there, it caused issues with deployStartedAt, might do the same with deployedAt. I am removing it as I think io-ts doesn't support optional props but for future reference we might reconsider using io-ts library

Comment (iain-b) [reply src/types.ts]: You can do this https://github.com/gcanti/io-ts/blob/master/Decoder.md#the-intersect-combinator to have optional properties but I agree it's really ugly for what is quite a common use case. I agree that we might reconsider.

Comment (Irma12) [reply src/types.ts]: I did follow that pattern but it still has weird behaviour when handling with undefined values. I did however found a way with using _union_. Maybe that would be good for now?

**Review by iain-b — APPROVED** [review]

**Pull request merged**